app/services/imports/organisation_la_import_service.rb

```diff
@@ -0,0 +1,23 @@
+module Imports
+  class OrganisationLaImportService < ImportService
+    def create_organisation_las(folder)
+      import_from(folder, :create_organisation_la)
+    end
+
+  private
+
+    def create_organisation_la(xml_document)
+      xml_document.remove_namespaces!
+      organisation = Organisation.find_by(old_org_id: record_field_value(xml_document, "InstitutionId"))
+
+      OrganisationLa.create!(
+        organisation:,
+        ons_code: record_field_value(xml_document, "ONSCode"),
+      )
+    end
+
+    def record_field_value(xml_document, field)
+      xml_document.at_xpath("//#{field}")&.text
```

Comment:
Maybe a method we could add at the top level? (ImportService)
Currently, the other method we have needs a namespace but there is value in having both methods.